common/agents/tasks/perform-forge-review.md

```diff
@@ -0,0 +1,434 @@
+
+Examples:
+- `AI: *Important*: This will panic on empty input`
+- `AI: Consider using iterators here`
+- `AI: (low) Rename to follow naming convention`
+- `AI: **Question**: Is this intentional?`
+
+**Filtering by priority:**
+
+If user instructions specify a priority filter (e.g., "important only"), create
+inline comments only for that priority level. Summarize the most relevant
+normal/low priority items in a `<details>` section in the review body:
+
+```markdown
+<details>
+<summary>Additional suggestions (normal/low priority)</summary>
+
+- `src/lib.rs:42` — Consider using iterators
+- `src/main.rs:15` — Minor: rename to follow convention
+</details>
+```
+
+Avoid inline comments for purely positive observations (e.g., "Good approach here").
+These create noise and require manual resolution. If positive aspects are worth
+noting, briefly mention them in the review body.
+
+**Review body content:**
+
+Do not summarize or re-describe the PR changes — the commit messages already
+contain that information. The review body should only contain:
+- Attribution header (required)
+- Positive observations worth highlighting (optional, brief)
+- Concerns not tied to specific lines (optional)
+- Notes about missing context in the PR description (if any)
+- Collapsed lower-priority items when filtering (see above)
+
+---
+
+## Workflow
+
+**Note**: If you already have a pending review on this PR, you cannot create
+another. Check for existing pending reviews first (see API Reference) and
+either add comments to it or delete it before proceeding.
+
+### Step 1: Check Out the PR
+
+Check out the PR branch locally. This lets you read files directly to get
+accurate line numbers (diff line numbers are error-prone):
+
+```bash
+# GitHub
+gh pr checkout PR_NUMBER
+
+# GitLab
+glab mr checkout MR_IID
+
+# Forgejo (using forgejo-cli, or fall back to git fetch)
+forgejo-cli pr checkout PR_INDEX
+# or: git fetch origin pull/PR_INDEX/head:pr-PR_INDEX && git checkout pr-PR_INDEX
+```
+
+### Step 2: See the Code
+
+After checkout, determine the merge base (fork point) and review the changes:
+
+```bash
+# Find the merge base with the target branch (usually main)
+MERGE_BASE=$(git merge-base HEAD main)
+
+# View commit history since fork point
+git log --oneline $MERGE_BASE..HEAD
+
+# View the combined diff of all changes
+git diff $MERGE_BASE..HEAD
+
+# Or view each commit's diff separately
+git log -p $MERGE_BASE..HEAD
+```
+
+Review commit-by-commit to understand the logical structure of the changes.
+Pay attention to commit messages — they should explain the "why" behind each
+change. For larger PRs, reviewing each commit separately often provides better
+context than a single combined diff. Note any commits where the message doesn't
+match the code changes or where the reasoning is unclear.
+
+### Step 3: Build the Review
+
+The scripts in this task are located in the `scripts/` subdirectory relative to this
```

Comment:
AI: **Question**: The task references `scripts/forge-review-start.sh` as a relative path from the task file location. However, when an agent checks out a different repo that has synced `common/`, the path would be `common/agents/tasks/scripts/`. Should the documentation clarify how to locate these scripts, or should agents be expected to discover them?